### PR TITLE
Cleanup of the repository by removing unused files and updating the Readme

### PR DESCRIPTION
**Pull request by l-jost**

### CODE REVIEW

**Review by jbinz — APPROVED** [review]

File: firmware/st_nucleo_f4.cfg [hunk 1]
```diff
@@ -1,13 +0,0 @@
-# This is for all ST NUCLEO with any STM32F4. Known boards at the moment:
```
Comment (stojadin2701) [inline firmware/st_nucleo_f4.cfg:1]: this should stay

File: firmware/cats_rev1Pro.ioc [hunk 2]
```diff
@@ -1,411 +0,0 @@
-#MicroXplorer Configuration settings - do not modify
```
Comment (stojadin2701) [inline firmware/cats_rev1Pro.ioc:1]: an updated IOC should be uploaded

File: firmware/st_nucleo_f4.cfg (hunk 1 above)
Comment (stojadin2701) [reply firmware/st_nucleo_f4.cfg:1]: I think it's used by openocd

**Review by stojadin2701 — CHANGES_REQUESTED** [review]
wrote some comments

**Review by stojadin2701 — APPROVED** [review]

**Pull request merged**